### DISCUSSION TITLE
Ratio between pressures

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#28] [General]
For the pressure we need to take the same density? or i need to change with the dependence of r as we see $n\propto r^{-8.0}$.

i think that this could be necessary.

**Comment by will-henney** [comment]
I am sorry, but I do not understand what you mean.  Are you talking about the ratio between the pressure at the base of the photoevaporation flow and the external pressure (which in this case is ram pressure of wind)?     Which density are you talking about, and why do you think it should fall so steeply?
